examples/webgl_geometry_teapot.html

```diff
@@ -363,6 +368,14 @@
 
 			}
 
+			updateStats();
```

Comment:
I think that if user interaction triggers re-render, frame rate still matters because some heavy process may cause frame stuck, for example the web page doesn't respond to user's next interaction for some time, so maybe stats still needed?